var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -382,6 +383,7 @@ def get_paths(self):
         else:
             paths = {
                 "hip-path": self.spec.prefix,
+                "hipify-clang": rocm_prefix,
```

Comment:
This should be a separate check like for `bitcode`, and should do

```
if self.spec.satisfies("@5.4:"):
    paths["hipify-clang"] = self.spec["hipify-clang"].prefix
```
since this is the section for if it's not external

Reply:
I think this is now fixed.